WalletWasabi.Fluent/ViewModels/Wallets/Send/LabelSelectionViewModel.cs

```diff
@@ -167,10 +167,26 @@ private async Task<(bool, Pocket[] pockets)> TryGetBestKnownByRecipientPocketsWi
 			.Union(_hiddenIncludedPockets)
 			.ToArray();
 
-	public async Task ResetAsync(Pocket[] pockets)
+	public async Task ResetAsync(Pocket[] pockets, List<SmartCoin>? coinsToExclude = null)
 	{
 		_allPockets = pockets;
 
+		if (coinsToExclude != null)
+		{
+			var pocketsWithoutBusyCoins = new List<Pocket>();
+			var coinjoiningPockets = pockets.Where(pocket => pocket.Coins.Any(coinsToExclude.Contains)).ToArray();
+
+			foreach (var pocket in coinjoiningPockets)
+			{
+				pocketsWithoutBusyCoins.Add(new Pocket((pocket.Labels, new CoinsView(pocket.Coins.Except(coinsToExclude)))));
```

Comment:
I don't think so, because we only interested if the `pocketsWithoutExcludedCoins` is enough to make the payment or not. We check it with `IsPocketEnoughAsync`.  If it's empty or not enough, we do everything as on `master`.